docs/changelog.md

```diff
@@ -19,25 +19,45 @@ permalink: /changelog/
 
 * **browser-session**
   * Added a use case for exiting fullscreen mode.
+
   ```kotlin
   val sessionUseCases = SessionUseCases(sessionManager)
   if (isFullScreenMode) {
     sessionUseCases.exitFullscreen.invoke()
   }
   ```
+
+  * We also added a `FullScreenFeature` that manages fullscreen support.
+
+  ```kotlin
+  val fullScreenFeature = FullScreenFeature(sessionManaager, sessionUseCases) { enabled ->
+    if (enabled) {
+      // Make custom views hide.
+    } else {
+      // Make custom views unhide.
+    }
+  }
+
+  override fun onBackPressed() : Boolean {
+    // Handling back presses when in fullscreen mode
+    if (fullScreenFeature.onBackPressed()) {
```

Comment:
nit: This should just be `return fullScreenFeature.onBackPressed()` or add an `else` block?

Reply:
Merging this for now and I'll correct it in a separate PR because I don't want to re-build again.